troupon/troupon/urls.py

```diff
@@ -4,6 +4,7 @@
 import account 
 
 urlpatterns = [
+    url(r'^accounts/', include('allaccess.urls')),
```

Comment:
@chidi yep it can't be ..... fortunately it comes with an 's' account(s)
The Django-all-access modules uses it by default...... so I left it alone